docs/porting/overview.md

```diff
@@ -1,5 +1,72 @@
-# 🔧 Overview of porting process
+# Overview of the Porting Process
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
\ No newline at end of file
+By [Phillip Carter](https://github.com/cartermp)
+
+If you've got code running on .NET Framework you may be interested in running your code on .NET Core.  This article will cover evaluating if porting to .NET Core is reasonable for you, an overview of the porting process, and a list of the tools you may use to port.
+
+## Should you port?
+
+The first thing you need to do is evaluate if it makes sense to port .NET Framework code over to .NET Core in the first place.  .NET Core offers some attractive capabilities, such as:
+
+- Using ASP.NET Core to write great web apps and services which you can write and deploy on your OS of choice
+- Console applications that can be Ahead-of-time compiled into native code with minimal dependencies
+- The ability to deploy apps and services app-local and with different versions of .NET Core side-by-side
+- The ability to write a UWP app that will run on multiple Windows 10 devices
+
+Your code could be a good candidate for porting to .NET Core if...
+
+- Your code is a library and you want to expand its reach
+- You have a compelling business case to run code on more than just Windows
+- You want to take advantage of app-local deployment and build self-contained console applications
+- You have expertise on non-Windows platforms you wish to leverage
+- You're looking for an opportunity to implement an architectural change for your services (e.g. a monolithic service to microservices deployed to containers)
+- You love new technology and want to cut your teeth on .NET Core
+
+Your code may not be a good candidate for porting if...
+
+- Your code is heavily tied to a Windows GUI or depends on Windows-specific features
+- You have no business needs or expertise which would warrant running on more than Windows
+- You don't have a need for app-local deployment or self-contained console apps
+- You're happy with your code running on .NET Framework as it is today
+
+If it makes sense to port at this time, then read on!
+
+## Overview of the Porting Process
+
+The recommended process for porting follows a series of steps.  Here's a short overview of the process in general.  There will be other specific steps to take when porting libraries, apps and services.
+
+1. Identify and account for your third-party dependencies.
+
+   This will involve understanding what your 3rd party dependencies are, how you depend on them, how to see if they also run on .NET Core, and steps you can take if they don't.
+   
```

Comment:
Give some guidance on what to do if a library you depend on isn't available on Core. I like to tell people to consider, in order: contact the author, offer to help, fork if it's OSS, switch to another library fulfilling the same purpose, or, as a last resort, write your own replacement.